### ISSUE TITLE
 ES6: Declare a Read-Only Variable with the const Keyword uppercase constant

### ISSUE THREAD

**Issue by arnas** [freeCodeCamp/freeCodeCamp#19037]
**Describe your problem and how to reproduce it:**
I find strange that tutorial suggest to name use upper case for const.  I agree that const should be uppercase if it's in application scope. In block or function scope camelCase makes more sense in my opinion.

**Add a Link to the page with the problem:**
https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/es6/declare-a-read-only-variable-with-the-const-keyword

**Comment by ezioda004** [comment]
Using uppercase is the convention and best practice when declaring `const`. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#Examples) also uses this convention.

**Comment by arnas** [comment]
In most cases yes, but in reality I would argue that this https://www.reddit.com/r/learnjavascript/comments/5ns7hq/constants_can_be_declared_with_uppercase_or/dce1s76 is more realistic.

**Comment by ezioda004** [comment]
Speaking from experience, const primitives are used with uppercase and objects are used with camel cases and I think that reddit thread also follows the same logic:
```js
const INC = "INCREMENT"; //String, can't be mutated
const someArr = [1, 2, 3, 4]; //Object, can be mutated
const PI = 3.14; //Number, can't be mutated
```
The challenges also use the same convention and therefore the challenge's text is completely fine.
The Redux makes use of this convention and I think it's best to keep this way so for beginner programmer ergonomics.

**Comment by nathanhannig** [comment]
I agree, const should be uppercase unless it is an object that mutates.
